src/main/scala/mesosphere/marathon/core/health/HealthCheck.scala

```diff
@@ -242,13 +244,17 @@ case class MesosHttpHealthCheck(
   port: Option[Int] = HealthCheckWithPort.DefaultPort,
   path: Option[String] = MarathonHttpHealthCheck.DefaultPath,
   protocol: Protocol = MesosHttpHealthCheck.DefaultProtocol,
+  ipProtocol: Option[IpProtocol] = None,
```

Comment:
I thought about that too, but since mesos protos are backwards compatible `ipProtocol` is defined as `optional`. I guess we can either omit setting it or adhere to the provided default.

Reply:
The Marathon model doesn't need to reflect the proto. If `None` equals `IPv4` and doesn't lead to problems elsewhere, I would make it explicit and make it mandatory. This effectively reduces the number of combinations we need to test.